whylog/tests/test_constraints/__init__.py

```diff
@@ -0,0 +1,55 @@
+from unittest import TestCase
+
+from whylog.constraints import IdenticalConstraint, TimeConstraint
+from whylog.constraints.exceptions import ConstructorGroupsError, ConstructorParamsError
+
+
+class TestIdenticalConstraint(TestCase):
```

Comment:
You can add tests for corner cases for example constructor takes to little or to much groups or wrong param dict and e.t.c. I would be not happy if I faced this bugs while writing plugin.